front/src/common/api/generatedEditoastApi.ts

```diff
@@ -4608,6 +4407,63 @@ export type RoundTrips = {
   /** List of round trips, each represented by a tuple */
   round_trips?: number[][];
 };
+export type Scenario = {
+  creation_date: string;
+  description: string;
+  electrical_profile_set_id?: number;
+  id: number;
+  infra_id: number;
+  last_modification: string;
+  name: string;
+  study_id: number;
+  tags: Tags;
+  timetable_id: number;
+};
+export type ScenarioWithDetails = Scenario & {
+  infra_name: string;
+  paced_trains_count: number;
+  trains_count: number;
+};
+export type ScenarioCreateForm = {
+  description?: string;
+  electrical_profile_set_id?: number | null;
+  infra_id: number;
+  name: string;
+  study_id: number;
```

Comment:
It's a detail, but would it be possible to have the study_id outside the `scenarioCreateForm` ? Same for the `studyCreateForm` and `project_id`. I don't know what is the more standard between putting it in the body but next to the form, or put it in the params though